### PR TITLE
Lame and Twolame should be moved to userland

### PR DESCRIPTION
**Pull request by alarcher**
Need to work on obsoletion in encumbered.

### CODE REVIEW

**Review by xen0l — APPROVED** [review]

**Comment by alarcher** [comment]
@pyhalov Could tell me if this is the right approach? I modified the history.mk to ignore dummy.p5m, the rest is the same.

**Comment by alarcher** [comment]
@pyhalov Should the component in userland be one more than the obsolete version in encumbered?

File: components/encumbered/meta-packages/history/Makefile [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#
+# CDDL HEADER END
+#
+# Copyright (c) 2011, 2015, Oracle and/or its affiliates. All rights reserved.
+#
+
+include ../../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		"Package History (encumbered)"
+COMPONENT_BUGDB=	gate-stuff/userland-encumbered
```
Comment (pyhalov) [inline components/encumbered/meta-packages/history/Makefile]: Remove BUGDB

Comment (pyhalov) [inline components/encumbered/meta-packages/history/Makefile]: Remove BUGDB

**Comment by pyhalov** [comment]
yes, userland version should be > obsolete version in encumbered.

File: components/meta-packages/history/Makefile [hunk 2]
```diff
@@ -23,89 +23,9 @@
...
-
-# For gmake clean to work
-COMPONENT_SRC=nonexistent
-
-PKGSEND =	/usr/bin/pkgsend
-PKGLINT =	/usr/bin/pkglint
-PKGMOGRIFY =	/usr/bin/pkgmogrify
-
 COMPONENT_NAME=		"Package History"
 COMPONENT_BUGDB=	gate-stuff/userland
```
Comment (pyhalov) [inline components/meta-packages/history/Makefile]: please, remove BUGDB

**Comment by alarcher** [comment]
@pyhalov Fixed.

**Comment by pyhalov** [comment]
@alarcher: You'll have to fix https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/tools/bass-o-matic#L128  to build history for encumbered at any change (as history changes will not touch Makefile).

**Pull request merged**